src/OrchardCore/OrchardCore.Users.Core/Models/User.cs

```diff
@@ -25,6 +27,8 @@ public class User : Entity, IUser
 
         public bool EmailConfirmed { get; set; }
 
+        public bool PhoneNumberConfirmed { get; set; }
```

Comment:
Why we need to confirm the phone?

Reply:
Because this is something we need to store for many reasons. When use SMS for 2FA.